scene/debugger/scene_debugger.cpp

```diff
@@ -552,32 +552,48 @@ SceneDebuggerTree::SceneDebuggerTree(Node *p_root) {
 		Node *n = stack[0];
 		stack.pop_front();
 		int count = n->get_child_count();
-		nodes.push_back(RemoteNode(count, n->get_name(), n->get_class(), n->get_instance_id()));
+
+		Dictionary extra_data = _get_extra_data(n);
+
+		nodes.push_back(RemoteNode(count, n->get_name(), n->get_class(), n->get_instance_id(), extra_data));
 		for (int i = 0; i < count; i++) {
 			stack.push_front(n->get_child(count - i - 1));
 		}
 	}
 }
 
+Dictionary SceneDebuggerTree::_get_extra_data(Node *p_node) {
+	Dictionary extra_data;
+	if (p_node->has_method(SNAME("is_visible_in_tree"))) {
+		extra_data["visible"] = p_node->call(SNAME("is_visible_in_tree"));
```

Comment:
This makes sub-nodes incorrectly display closed eye icon if the parent is invisible. It's rather confusing and inconsistent. I think using `is_visible` would be fine (and would also handle CanvasLayers).